gui/tools/RunAllTool.cxx

```diff
@@ -51,13 +51,8 @@
 class RunAllToolPrivate
 {
 public:
-  RunAllToolPrivate()
-    : output(0), failed(false)
-  {
-  }
-
   AbstractTool::Outputs output;
```

Comment:
Does this initialize to `0` automatically?  (not sure what the type is without digging into the code)

Reply:
Yes; it's a QFlags. Initializing QFlags with `nullptr` is deprecated; the default constructor should be used instead. (In fact, this initializer never should have been present for that reason. The reason `= 0` was previously supported was for default function parameters, because before C++11 you otherwise had to spell out the type name twice. It was never meant to be used as a member initializer because there was never a reason to do so.)